ui/tests/pages/components/console/ui-panel.js

```diff
@@ -54,12 +54,18 @@ export default {
     eventProperties: { keyCode: keys.ENTER },
   }),
   hasInput: isPresent('[data-test-component="console/command-input"] input'),
-  runCommands: async function (commands) {
+  runCommands: async function (commands, shouldToggle = true) {
```

Comment:
We needed to add an extra condition here, because there are tests that don't actually need the toggle to happen. Maybe it's rendering the console/ui-panel directly, or maybe it's a test that runs multiple commands and multiple toggles would cause those tests to fail.